samples/common/index.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Package index for common sample pieces.
+ */
+
+'use strict';
+
+module.exports = {
+  ...require('./common'),
```

Comment:
stylistically, I'd probably just make the contents of `common.js`, `index.js`, when there's only a single export in a folder.

Reply:
I'm also removing common.js and its index.js, so this is going away.